src/commands/projects/add-pull-request.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * This command adds a pull request to a GitHub project column.
+ *
+ * @see: https://octokit.github.io/rest.js/#api-Projects-createCard
+ * const result = await octokit.projects.createCard({column_id, note, content_id, content_type})
+ * /projects/columns/:column_id/cards
+ */
+const flow = require('lodash.flow')
+
+const commonYargs = require('../../../lib/common-yargs')
+const printOutput = require('../../../lib/print-output')
+const authenticatedOctokit = require('../../../lib/octokit')
+
+/**
+ * yargs builder function.
+ *
+ * @param {import('yargs').Yargs} yargs - Instance of yargs
+ */
+const builder = yargs => {
+	const baseOptions = flow([
+		commonYargs.withToken,
+		commonYargs.withJson,
+	])
+	return baseOptions(yargs)
+		.option('column_id', {
+			describe: 'Project column ID',
+			demandOption: true,
+			type: 'number'
+		})
+		.option('pull_request_id', {
+			describe: 'Pull request ID',
+			demandOption: true,
+			type: 'number'
+		});
+}
+
+/**
+ * Add a pull request to a GitHub project column.
+ *
+ * @param {object} argv - argv parsed and filtered by yargs
+ * @param {string} argv.token
+ * @param {string} argv.json
+ * @param {number} argv.column_id
+ * @param {number} argv.pull_request_id
+ * @throws {Error} - Throws an error if any required properties are invalid
+ */
+const handler = async ({ token, json, column_id, pull_request_id }) => {
+
+	// Ensure that all required properties have values
+	const requiredProperties = {
+		column_id,
+		pull_request_id,
+	}
+	if (Object.values(requiredProperties).some(property => !property)) {
+		throw new Error(`Please provide all required properties: ${Object.keys(requiredProperties).join(', ')}`)
+	}
```

Comment:
This is validation being done inside the handler, when yargs promotes dealing with validation using it's API. We should avoid putting this logic in the handlers.

0bf9d50 introduced a good use of the `check` function for custom validation, could we not continue to use that?